cmd/spx/go.mod

```diff
@@ -22,8 +22,6 @@ require (
 
 replace github.com/goplus/spx => ../../
 
+replace github.com/realdream-ai/gdspx/cmd/gdspx => github.com/realdream-ai/gdspx/cmd/gdspx v0.0.0-20241028072423-3577d7e1307f
```

Comment:
Nit: We can declare the dep version directly

```
require (
	github.com/realdream-ai/gdspx/cmd/gdspx v0.0.0-20241028072423-3577d7e1307f
)
```

instead of using `replace`

```
require (
	github.com/realdream-ai/gdspx/cmd/gdspx v0.0.0
)
replace github.com/realdream-ai/gdspx/cmd/gdspx => github.com/realdream-ai/gdspx/cmd/gdspx v0.0.0-20241028072423-3577d7e1307f
```